src/20240306_AaveV3Ethereum_AaveLiquidityCommiteeFunding/AaveV3Ethereum_AaveLiquidityCommiteeFunding_20240306.t.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {MiscEthereum} from 'aave-address-book/MiscEthereum.sol';
+import {ProtocolV3TestBase} from 'aave-helpers/ProtocolV3TestBase.sol';
+
+import {AaveV3Ethereum_AaveLiquidityCommiteeFunding_20240306} from './AaveV3Ethereum_AaveLiquidityCommiteeFunding_20240306.sol';
+
+/**
+ * @dev Test for AaveV3Ethereum_AaveLiquidityCommiteeFunding_20240306
+ * command: make test-contract filter=AaveV3Ethereum_AaveLiquidityCommiteeFunding_20240306
+ */
+contract AaveV3Ethereum_AaveLiquidityCommiteeFunding_20240306_Test is ProtocolV3TestBase {
+  event SwapRequested(
+    address milkman,
+    address indexed fromToken,
+    address indexed toToken,
+    address fromOracle,
+    address toOracle,
+    uint256 amount,
+    address indexed recipient,
+    uint256 slippage
+  );
+
+  event Transfer(address indexed from, address indexed to, uint256 value);
+
+  AaveV3Ethereum_AaveLiquidityCommiteeFunding_20240306 internal proposal;
+
+  address swapper;
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 19392386);
```

Comment:
I would update to the latest test block as the GHO balance of the collector has been changed